api/models/call.go

```diff
@@ -149,6 +149,12 @@ type Call struct {
 
 	// App this call belongs to.
 	AppID string `json:"app_id" db:"app_id"`
+
+	// App this call belongs to.
+	TriggerID string `json:"trigger_id" db:"trigger_id"`
+
+	// App this call belongs to.
```

Comment:
again

Reply:
Fixed (and actually started populating these, which I'd missed)